CONTRIBUTING.md

```diff
@@ -19,16 +19,56 @@ When opening an issue, please select the most appropriate template for what you'
 
 ### Getting Started
 
-Follow the instructions to [setup your dev environment to build copa](./docs/tutorials/dev-setup.md).
+Follow the instructions to [set up your dev environment to build Copacetic](#IDE-setup).
 
-For an overview of the project components, refer to the [copa design](./docs/vulnerability-driven-patching.md) document.
+For an overview of the project components, refer to the [Copa design](./website/docs/design.md) document.
+
+### IDE Setup
+
+We recommend using Goland as the dev environment to simplify the setup process and have access to various debugging tools.
+
+* Download and install [Jetbrains Toolbox](https://www.jetbrains.com/toolbox-app/). 
```

Comment:
Can we make this generic instead of a recommending a paid IDE/tool?

Reply:
if we need IDE specific instructions, we should link to those as those might change over time. Copa docs should not be over optimized on a specific setup.

Reply:
I've made all of these changes locally but I noticed the [website's contributing docs](https://project-copacetic.github.io/copacetic/website/contributing) mention a VS Code dev container. Did we want to keep this in for future developers? Or should we remove it to stay in line with not being overly optimized for a specific setup?